src/test/java/nl/tudelft/contextproject/logging/TestLog.java

```diff
@@ -280,10 +269,8 @@ public void testCreateLogLevel() {
 	 */
 	@Test
 	public void testShutdown() {
-		//Shutdown the log
 		Log.shutdown();
-		
-		//A file handler is closed, when isLoggable returns false for messages that should otherwise be logged.
```

Comment:
This comment explains why isLoggable is an appropriate way of testing if SHUTDOWN works, so should be readded.

Without this comment, it would seem like the test is simply for coverage only.